core/storage/translation/gae_models.py

```diff
@@ -34,6 +38,181 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+# TODO(#14537): This is a duplicate of the same TypedDict in translation_domain,
+# it should be removed after we refactor our importing.
+class TranslatedContentDict(TypedDict):
+    """Dictionary representing TranslatedContent object."""
+
+    content: Union[str, List[str]]
+    needs_update: bool
+
+
+# TODO(#14537): This is a duplicate of the same enum in translation_domain,
+# it should be removed after we refactor our importing.
+class EntityTypesSupportingNewTranslationArch(enum.Enum):
+    """Represents all possible entity types which support new translations
+    architecture.
+    """
+
+    ENTITY_TYPE_EXPLORATION = 'exploration'
+    ENTITY_TYPE_QUESTION = 'question'
+
+
+class EntityTranslationsModel(base_models.BaseModel):
+    """Model for storing entity translations ."""
+
+    # The id of the corresponding entity.
+    entity_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The type of the corresponding entity, supporting new translation
+    # architecture.
+    entity_type = datastore_services.StringProperty(
+        required=True, indexed=True, choices=[
+            feconf.ENTITY_TYPE_EXPLORATION,
+            feconf.ENTITY_TYPE_QUESTION
+            ])
```

Comment:
Deindent by 4 to match opening bracket indentation.

Reply:
Updated.